packages/cli/api-importers/openapi/openapi-ir-to-fern/src/OpenApiIrConverterContext.ts

```diff
@@ -34,19 +34,19 @@ export class OpenApiIrConverterContext {
     public environmentOverrides: RawSchemas.WithEnvironmentsSchema | undefined;
     public authOverrides: RawSchemas.WithAuthSchema | undefined;
     public globalHeaderOverrides: RawSchemas.WithHeadersSchema | undefined;
-    public detectGlobalHeaders: boolean;
-    public objectQueryParameters: boolean;
-    public respectReadonlySchemas: boolean;
-    public respectNullableSchemas: boolean;
-    public onlyIncludeReferencedSchemas: boolean;
-    public inlinePathParameters: boolean;
-    public useBytesForBinaryResponse: boolean;
-    public respectForwardCompatibleEnums: boolean;
-    public wrapReferencesToNullableInOptional: boolean;
-    public coerceOptionalSchemasToNullable: boolean;
-    public groupEnvironmentsByHost: boolean;
-
-    private enableUniqueErrorsPerEndpoint: boolean;
+    public detectGlobalHeaders!: boolean;
+    public objectQueryParameters!: boolean;
+    public respectReadonlySchemas!: boolean;
+    public respectNullableSchemas!: boolean;
+    public onlyIncludeReferencedSchemas!: boolean;
+    public inlinePathParameters!: boolean;
+    public useBytesForBinaryResponse!: boolean;
+    public respectForwardCompatibleEnums!: boolean;
+    public wrapReferencesToNullableInOptional!: boolean;
+    public coerceOptionalSchemasToNullable!: boolean;
+    public groupEnvironmentsByHost!: boolean;
+
+    private enableUniqueErrorsPerEndpoint!: boolean;
```

Comment:
Using `!` is problematic. Instead storing all these flags here, expose the options object and have consumers grab it from the nested options property.